SteamKit2/SteamKit2/Util/HardwareUtils.cs

```diff
@@ -56,17 +56,21 @@ public override byte[] GetMacAddress()
             // mono seems to have a pretty solid implementation of NetworkInterface for our platforms
             // if it turns out to be buggy we can always roll our own and poke into /sys/class/net on nix
 
-            var firstEth = NetworkInterface.GetAllNetworkInterfaces()
-                .Where( i => i.NetworkInterfaceType == NetworkInterfaceType.Ethernet || i.NetworkInterfaceType == NetworkInterfaceType.Wireless80211 )
-                .FirstOrDefault();
+            try {
```

Comment:
Nit: The style used across SteamKit is to have the opening curly brace on a new line. I see you've done that for the `if` block, but not for `try` or `catch`. Can you please fix these?

Reply:
Will do!